src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -222,10 +222,10 @@
 		<type fullname="System.Int64">
 			<field name="m_value"/>
 		</type>
-
+    
```

Comment:
Trailing whitespace.

Reply:
there are whitespace issues in general in this PR, mixing spaces instead of tabs.

Reply:
If possible we should have `editor.config` for this, otherwise it's easy to break this (I honestly didn't notice anything when looking at the diff in VS).

Fixed now.